src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Filters/LiquidViewFilters.cs

```diff
@@ -124,7 +124,8 @@ static async ValueTask<FluidValue> Awaited(Task<IHtmlContent> task)
                 return new HtmlContentValue(await task);
             }
 
-            if (input.ToObjectValue() is IShape shape)
+            var shape = input.ToObjectValue();
+            if (shape != null)
```

Comment:
I would suggest to check if it is an `IShape` or an `IHtmlContent`

But i saw that it is checked in `IHtmlContent CoerceHtmlString(object value)`

And if it is another object type, we return `StringHtmlContent(value.ToString())`

So that's okay ;)